docs/Flask code example/Flask Documentation.md

```diff
@@ -0,0 +1,62 @@
+# Sending Request to MeowForms😸 (Flask)
+
+⚠️Make sure you have registed to the MeowForms😸
+
+The endpoint to submit a response will be in the format:
+```
+https://server.meowform.xyz/form/<your_registered_email>&<your_form_name>
+```
+along with the data from the HTML form created by the user!
+
+If the POST request is made sucessfully the endpoint will return status 200 😁 otherwise it will result in some error 😕
+
+We will make use of the ```requests``` module in Python to work with HTTPS methods.<br><br>
```

Comment:
hey these br tags are causing a problem in build either remove them or close them :)

Reply:
will do it by today..

Reply:
sure :)